tests/ted.json

```diff
@@ -1,6 +1,6 @@
 {
-  "libmfx_folder": "$MFX_HOME/__cmake/intel64.make.release/__bin/release",
-  "samples_folder": "$MFX_HOME/__cmake/intel64.make.release/__bin/release",
+  "libmfx_folder": "$MFX_HOME/build/__bin/release",
```

Comment:
using `build` is actually a convention that user will build mediasdk as follows:
```
mkdir build && cd build
cmake <...> ..
```

Also, why do we actually need a path to non-installed medisdk binaries. So, what do we actually test: installed mediasdk version or non-installed one?

Reply:
We test non-installed binaries (test system copies them from "libmfx_folder").